services/catalog/service/mosaic-hosting-deployment-manifest.yaml

```diff
@@ -30,6 +30,7 @@ regularVariables:
   DATABASE_LOGIN: '${__ax_hosted__.pg.db_login_role}'
   DATABASE_LOGIN_PASSWORD: '${__ax_hosted__.pg.db_login_password}'
   DATABASE_GQL_ROLE: '${__ax_hosted__.pg.db_gql_role}'
+  PG_POOL_MAX_CONNECTIONS: '${__ax_hosted__.pg.pool_max_connections}'
```

Comment:
Had a chat with @EmperorRXF as well, and we can remove the placeholder here and get the media-template developers to straightaway mention a value, since it's something they can decide and the Hosting Service does not have to interfere with it. I shall push the fix shortly. :)